app/Http/Controllers/FileController.php

```diff
@@ -21,20 +21,20 @@
 
 class FileController extends Controller{
```

Comment:
The file contains no modification but spaces and tabs. Please remove from commit